scaladoc-js/common/resources/code-snippets.css

```diff
@@ -112,16 +112,18 @@ input:checked + .slider:before {
   transform: translateX(var(--translation-size));
 }
```

Comment:
Not really important, but the concept of “resources” does not really make sense in Scala.js projects. I suggest moving this file to a `src/main/css` directory.

Reply:
Oh, ok